clang/test/SemaHIP/amdgpu-builtin-in-lambda.hip

```diff
@@ -0,0 +1,24 @@
+// REQUIRES: amdgpu-registered-target
```

Comment:
Don't think it actually does

Reply:
I've just checked using experimental target `csky-unknown-elf ` that's not enabled by default, and clang indeed errors out if we attempt to generate code, but works OK with `-fsyntax-only`.

Reply:
-emit-llvm should work fine, real codegen will obviously fail